### PR TITLE
Visual fixes

### PR DESCRIPTION
**Pull request by lbourdois**
Hi @lewtun 
The propagation of points from https://github.com/huggingface/course/pull/412 to English.

### CODE REVIEW

File: chapters/en/chapter1/4.mdx [hunk 1]
```diff
@@ -81,6 +81,11 @@ Imagine if each time a research team, a student organization, or a company wante
 
 This is why sharing language models is paramount: sharing the trained weights and building on top of already trained weights reduces the overall compute cost and carbon footprint of the community.
 
+By the way, you can evaluate the carbon footprint of your models' training through several tools. For example [ML CO2 Impact](https://mlco2.github.io/impact/) or [Code Carbon]( https://codecarbon.io/) which is integrated in 🤗 Transformers. To learn more about this, you can read this [blog post](https://huggingface.co/blog/carbon-emissions-on-the-hub) which will show you how to generate an `emissions.csv` file with an estimate of the footprint of your training, as well as the [documentation](https://huggingface.co/docs/hub/model-cards-co2) of 🤗 Transformers addressing this topic.
+
+
+Translated with www.DeepL.com/Translator (free version)
```
Comment (lewtun) [inline chapters/en/chapter1/4.mdx:87]: DeepL is awesome, but let's remove this :)
```suggestion
```

**Review by lewtun — APPROVED** [review]
Thanks for these fixes @lbourdois ! I've left one nit, but then we can merge 🔥

File: chapters/en/chapter1/4.mdx (hunk 1 above)
Comment (lbourdois) [reply chapters/en/chapter1/4.mdx]: Oops 😁 
Fixed

**Comment by lewtun** [comment]
Thanks for the fix!

**Pull request merged**